pdc/apps/common/viewsets.py

```diff
@@ -205,7 +205,7 @@ def _check_ordering_keys(self, request):
         serializer_fields = self._get_fields_from_serializer_class()
         valid_fields = list(set(model_fields).intersection(set(serializer_fields)))
         valid_fields += self.queryset.query.aggregates.keys()
-        tmp_list = [param.strip().lstrip('-') for param in ordering_keys.split(',')]
+        tmp_list = [param.strip().lstrip('-').split("__")[0] for param in ordering_keys.split(',')]
```

Comment:
I am wondering if we can skip nested ordering check here and let is_valid_field do the check?
I am not sure